src/pages/WordGroup/WordEditor.tsx

```diff
@@ -44,7 +44,7 @@ const defaultValues = { text: "", audio: "", example_sentence: "" };
 const WordEditor = ({ classes, match, values = defaultValues }: Props) => {
   const { t } = useTranslation();
 
-  // TODO: Unfortunately, react-apollo-hooks doesn't support yet the error, loading object in mutations (unlike with query...)
+  // TODO: Unfortunately, @apollo/react-hooks doesn't support yet the error, loading object in mutations (unlike with query...)
```

Comment:
This is still true?

Reply:
Oh sorry - That comment got changed because of the search and replace operation was not intended 😄. And to your question: No, that is finally changed (In fact I think it already has changed in the other lib recently), we can now use for example: 
```
  const [createComponent, { loading: createLoading, error: createError }] = useMutation(CREATE_COMPONENT);
```
Note the usage of the rename operation in the destructuring. This way you can have multiple mutation in one functional component without name clashes...